internal/lsp/server.go

```diff
@@ -84,7 +88,8 @@ type Server struct {
 	diagsMu sync.Mutex
 	diags   map[string][]protocol.Diagnostic
 
-	testCtrl *TestController
+	testCtrl     *TestController
+	serverConfig ServerConfig
```

Comment:
I find `Server.serverConfig` is too repetetive. How about:
```suggestion
	Config
```

Reply:
`Config`as a field name clashes with `Config`method